Mathlib/Topology/Algebra/Constructions.lean

```diff
@@ -180,4 +180,24 @@ theorem continuous_coe_inv : Continuous (fun u => ↑u⁻¹ : Mˣ → M) :=
 #align units.continuous_coe_inv Units.continuous_coe_inv
 #align add_units.continuous_coe_neg AddUnits.continuous_coe_neg
 
+@[to_additive]
+protected lemma continuous_inv : Continuous (fun u ↦ u⁻¹ : Mˣ → Mˣ) := by
+  rw [Units.continuous_iff]
+  exact ⟨continuous_coe_inv, continuous_val⟩
+
+/-- The inversion map on `Units` is a homeomorphism. -/
+@[to_additive (attr := simps!) "The negation map on `AddUnits` is a homeomorphism."]
+protected def homeomorph_inv : Mˣ ≃ₜ Mˣ :=
+  { left_inv := fun _ ↦ rfl,
+    right_inv := fun _ ↦ rfl,
+    continuous_toFun := Units.continuous_inv,
+    continuous_invFun := Units.continuous_inv }
+
+lemma inducing_coe_iff : Inducing (coeHom M) ↔ Inducing (fun u ↦ ↑u⁻¹ : Mˣ → M) :=
```

Comment:
Why not use `Units.val` instead of `coeHom` in the LHS? Also, LHS is simpler than RHS. I suggest
```suggestion
lemma inducing_val_inv_iff : Inducing (fun u ↦ ↑u⁻¹ : Mˣ → M) ↔ Inducing ((↑) : Mˣ → M) :=
```

Reply:
In fact, I think that we should have `Homeomorph.inducing_comp_left_iff` and `Homeomorph.embedding_comp_left_iff`.